README.md

```diff
@@ -28,38 +28,48 @@ The npm installs the binaries as both `raml-toolkit` and `ramlint` and they can
 **Note:** Some commands require environment variables to be set. This can be done using a [.env file](https://www.npmjs.com/package/dotenv#rules) in your working directory (the directory from which you run `raml-toolkit`).
 
 <!-- commands -->
-- [`raml-toolkit diff APISPECBASEPATH APISPECNEWPATH`](#raml-toolkit-diff-apispecbasepath-apispecnewpath)
+- [`raml-toolkit diff BASE NEW`](#raml-toolkit-diff-base-new)
```

Comment:
I didn't mean literally base and new. I mean to have base and new in place of old and new, something like BASEAPI NEWAPI or BASEAPISPEC NEWAPISPEC

Sorry for not being clear

Reply:
Because oclif makes the names caps, I think that BASE / NEW are the cleanest for readability.